dist/svg.js

```diff
@@ -1694,20 +1704,53 @@ SVG.FX = SVG.invent({
 
       // convert current time to position
       if(!ignoreTime) this.pos = this.timeToPos(+new Date)
+      else {
```

Comment:
while
```js
if (...) {

} else something()
```

is good readable, this one here confuses me a lot.
Better add the brackets in this case